editor/editor_node.cpp

```diff
@@ -5026,10 +5026,13 @@ String EditorNode::_get_system_info() const {
 		rendering_method = "Mobile";
 	} else if (rendering_method == "gl_compatibility") {
 		rendering_method = "Compatibility";
-		driver_name = GLOBAL_GET("rendering/gl_compatibility/driver");
 	}
 	if (driver_name == "vulkan") {
 		driver_name = "Vulkan";
+	} else if (driver_name == "d3d12") {
+		driver_name = "Direct3D 12";
+	} else if (driver_name == "opengl3_angle") {
```

Comment:
If we're making a special case for `opengl3_angle`, we might as well do one for `opengl3_es` too.

Reply:
I guess it this case, we should change `opengl3` to return `OpenGL 3` instead of `GLES3`.

Reply:
Updated to return `OpenGL 3` or `OpenGL ES 3` based on actually used backend.